lib/connection/connection_config.js

```diff
@@ -484,6 +485,13 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     }
   }
 
+  let includeRetryReason = true;
+  if (Util.exists(options.includeRetryReason)) {
+    Errors.checkArgumentValid(Util.isBoolean(options.includeRetryReason),
+      ErrorCodes.ERR_CONN_CREATE_INVALID_INCLUDE_RETRY_REASON);
+      includeRetryReason = options.includeRetryReason;
```

Comment:
this line does not seems to be properly indented

Reply:
added

Reply:
i don't think it is fixed

Reply:
Interesting... I can see there is an indent between includRetryReason and ErrorCode check in the file changes.

Reply:
Double checked, and the space is applied.

Reply:
I double checked and there is a space

Reply:
when I said indent I meant not an empty line but the amount of spaces from the left side :)

Reply:
Ahhh, I see. I edited it :).